rest-testing/src/test/java/com/baeldung/rest/cucumber/CucumberIntegrationTest.java

```diff
@@ -5,6 +5,6 @@
 import cucumber.api.junit.Cucumber;
 
 @RunWith(Cucumber.class)
-@CucumberOptions(features = "classpath:Feature")
+@CucumberOptions(features = "classpath:karate")
 public class CucumberIntegrationTest {
```

Comment:
Does this class do anything? Looks like maybe a leftover from an earlier commit.